src/NServiceBus.Core/TaskEx.cs

```diff
@@ -6,5 +6,11 @@ static class TaskEx
     {
         //TODO: remove when we update to 4.6 and can use Task.CompletedTask
         public static readonly Task Completed = Task.FromResult(0);
+
+        // ReSharper disable once UnusedParameter.Global
+        public static void Ignore(this Task task)
+        {
+            // Intentionally left blank
```

Comment:
Rename to `DoNotAwait()`?

-- 
*#NSBcon 2015 • The "All About NServiceBus" conference is back *
*• Learn more <http://go.particular.net/l/27442/2015-08-21/5d4wzp> →*